src/Server.cpp

```diff
@@ -47,7 +47,10 @@ bool	Server::isValueForKey(std::string location, std::string key, std::string va
 
 void	Server::setName(std::string serverName)
 {
-	this->_name = serverName;
+	if (serverName.empty())
+		this->_name = "default";
```

Comment:
we could probably have a definition for default server name, instead of the string literal. Not a big thing tho, so can be changes next time as well.

Reply:
you're right! done.